website/docs/blog.md

```diff
@@ -3,7 +3,112 @@ id: blog
 title: Blog
 ---
 
-_This section is a work in progress. [Welcoming PRs](https://github.com/facebook/docusaurus/issues/1640)._
+## Initial Setup
+
+To setup your site's blog, start by creating a `blog` directory within your repo's `my-website` directory.
+
+Then, add a navbar link to your blog within `docusaurus.config.js`:
+
+```js
+links: [
+    ...
+    {to: 'blog', label: 'Blog', position: 'left'}, // position: 'left' for creating Blog link on the left side of navbar, position: 'right' for creating Blog link on the right side of navbar
+    ...
+]
+```
+
+## Adding Posts
+
+To publish in the blog, create a file within the blog directory with a formatted name of `YYYY-MM-DD-my-blog-post-title.md`. The post date is extracted from the file name.
+
+For example, at `my-website/blog/2019-09-05-hello-docusaurus-v2.md`:
+
+```yml
+---
+id: hello-docusaurus-v2
+title: Welcome Docusaurus v2
+author: Dattatreya Tripathy
```

Comment:
I would love to credit your contributions on our release notes, but do you mind using a more neutral reference (i.e., you may use Docusaurus directly, the twitter handle is @docusaurus) here for the consistency of Docusaurus?

Reply:
Would make the required changes!!!